src/userGuide/UserGuideFi.tsx

```diff
@@ -495,8 +491,8 @@ function UserGuideFi(): ReactElement {
       >
         <p>
           Palvelut hyödyntävät Helsinki-profiilin hallinnoimia tietoja
-          ilmoittamallaan tavalla. Tunnistautumalla ensimmäisen kerran
-          palveluun, näet mitä tietoja palvelu hyödyntää.
+          ilmoittamallaan tavalla. Kirjautumalla ensimmäisen kerran palveluun,
```

Comment:
Pilkku pois. Kts. https://kielitoimistonohjepankki.fi/ohje/pilkku-ja-lauseenvastike-auringon-noustua-lahdimme/
```suggestion
          ilmoittamallaan tavalla. Kirjautumalla ensimmäisen kerran palveluun
```